src/main/java/com/github/dzieciou/testing/curl/Http2Curl.java

```diff
@@ -175,9 +175,7 @@ public Headers(List<Header> toProcess) {
     }
   }
 
-  @SuppressWarnings("deprecation")
-  private CurlCommand http2curl(HttpRequest request)
-      throws NoSuchFieldException, IllegalAccessException, IOException {
```

Comment:
Right, thanks for fixing that. In fact, the first two exceptions are wrapped in `RuntimeException` downstream, so it made no sense to declare them here.